### PR TITLE
Align Multiselect Text centered

### LINKED ISSUE #2938: Fix text alignment
**Current behaviour:**
If mulitselect is activated the "Nr selected" text is not aligned in the middle:

**Wanted behaviour:**
the "Nr selected" should be aligned in the same way as the arrow. 
(meaning that the text should have the central-axe as the arrow)

### PR DESCRIPTION
**Pull request by Elblinator**
closes #2938

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.html [hunk 1]
```diff
@@ -20,10 +20,9 @@ <h2>{{ 'Files' | translate }}</h2>
     </div>
 
     <!-- Multiselect info -->
-    <div *ngIf="isMultiSelect" class="central-info-slot">
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.html:23]: Is this intended?

File: client/src/app/site/pages/organization/pages/mediafiles/modules/organization-mediafile-list/components/organization-mediafile-list/organization-mediafile-list.component.html [hunk 2]
```diff
@@ -20,10 +20,9 @@ <h2>{{ 'Files' | translate }}</h2>
     </div>
 
     <!-- Multiselect info -->
-    <div *ngIf="isMultiSelect" class="central-info-slot">
+    <div class="central-info-slot">
```
Comment (bastianjoel) [inline client/src/app/site/pages/organization/pages/mediafiles/modules/organization-mediafile-list/components/organization-mediafile-list/organization-mediafile-list.component.html:23]: Is this intended?

File: client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.html (hunk 1 above)
Comment (Elblinator) [reply client/src/app/site/pages/meetings/pages/mediafiles/modules/mediafile-list/components/mediafile-list/mediafile-list.component.html:23]: Yes, 
I tested if this changes the OpenSlides behaviour and saw nothing
And I would assume that the code should ideally be as similar as possible in the

File: client/src/app/site/pages/organization/pages/mediafiles/modules/organization-mediafile-list/components/organization-mediafile-list/organization-mediafile-list.component.html (hunk 2 above)
Comment (Elblinator) [reply client/src/app/site/pages/organization/pages/mediafiles/modules/organization-mediafile-list/components/organization-mediafile-list/organization-mediafile-list.component.html:23]: same as above

**Review by bastianjoel — APPROVED** [review]

**Comment by MSoeb** [comment]
As discussed: in mobile view the text alignment doesn't work as it should.

**Review by MSoeb — CHANGES_REQUESTED** [review]
As discussed: On mobile view the text alignment isn't centered. This should be fixed.

**Review by bastianjoel — APPROVED** [review]

**Pull request merged**